plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/devToolsTab/nodes/CawsRootNode.kt

```diff
@@ -45,16 +45,21 @@ class CawsRootNode(private val nodeProject: Project) : AbstractTreeNode<String>(
             is ActiveConnection.ValidBearer -> {
                 when (connection.connectionType) {
                     ActiveConnectionType.BUILDER_ID -> message("caws.connected.builder_id")
-                    else -> message("caws.connected.identity_center")
+                    else -> message(
+                        "caws.connected.identity_center",
+                        connection.activeConnectionBearer?.let { "(${it.id})" } ?: ""
```

Comment:
We can probably trim the sso:session text and just have the session name

Reply:
I'm not sure all ids start with sso:session. Trimming just for this seems ad-hoc.

Reply:
Ah it looks it's added in https://github.com/aws/aws-toolkit-jetbrains/blob/649c45dffffcb8e657d629b049e3035b0602bd4a/plugins/toolkit/core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt#L140